client-templates/jira/.env.sample

```diff
@@ -0,0 +1,22 @@
+# your unique broker identifier
+BROKER_TOKEN=
+
+# your personal username to your Jira Server account
+JIRA_USERNAME=
+
+# your personal password to your Jira Server account
+JIRA_PASSWORD=
+
+# the host where your Jira Server is running, excluding scheme.
+# i.e. jira.yourdomain.com
+JIRA=
```

Comment:
@joshje maybe call it JIRA_URI instead of just JIRA?

Reply:
or JIRA_HOST

Reply:
I added `JIRA_API` as:
```bash
JIRA_API=$JIRA/rest/api/2
```

This follows the same convention as https://github.com/snyk/broker/blob/master/client-templates/bitbucket-server/.env.sample#L15:L18